modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -0,0 +1,94 @@
+ifndef::openshift-rosa[]
+{product-title} 
+endif::[]
+subscriptions, resource quotas and deployment scenario. For more information, contact your sales representative or Red Hat support.
+====
+
+.Prerequisites
+
+ifdef::openshift-rosa[]
+* You created a ROSA cluster.
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+* You created an {product-title} cluster.
+endif::[]
+
+.Procedure
+
+. From {cluster-manager-url}, navigate to the *Clusters* page and select your cluster.
+
+. Under the *Machine pools* tab, click *Add machine pool*.
+
+. Add a *Machine pool name*.
+
+. Select a *Worker node instance type* from the drop-down menu. The instance type defines the vCPU and memory allocation for each compute node in the machine pool.
+
+. Optional: Configure autoscaling for the machine pool:
+.. Select *Enable autoscaling* to automatically scale the number of machines in your machine pool to meet the deployment needs.
+.. Set the minimum and maximum node count limits for autoscaling. The cluster autoscaler does not reduce or increase the machine pool node count beyond the limits that you specify.
+** If you deployed your cluster using a single availability zone, set the *Minimum and maximum node count*. This defines the minimum and maximum compute node limits in the availability zone.
+** If you deployed your cluster using multiple availability zones, set the *Minimum nodes per zone* and *Maximum nodes per zone*. This defines the minimum and maximum compute node limits per zone.
++
+[NOTE]
+====
+Alternatively, you can set your autoscaling preferences for the machine pool after the machine pool is created.
+====
+
+. If you did not enable autoscaling, select a compute node count:
+* If you deployed your cluster using a single availability zone, select a *Worker node count* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool for the zone.
+* If you deployed your cluster using multiple availability zones, select a *Worker node count (per zone)* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool per zone.
+
+. Optional: Add node labels and taints for your machine pool:
+.. Expand the *Edit node labels and taints* menu.
+.. Under *Node labels*, add *Key* and *Value* entries for your node labels.
+.. Under *Taints*, add *Key* and *Value* entries for your taints.
+.. For each taint, select an *Effect* from the drop-down menu. Available options include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.
++
+[NOTE]
+====
+Alternatively, you can add the node labels and taints after you create the machine pool.
+====
+
+ifdef::openshift-dedicated[]
+. Optional: If you deployed {product-title} on AWS, select *Use Amazon EC2 Spot Instances* if you want to configure your machine pool to deploy machines as non-guaranteed AWS Spot Instances. For information, see link:https://aws.amazon.com/ec2/spot/[Amazon EC2 Spot Instances] in the AWS documentation.
```

Comment:
`Use Amazon EC2 Spot Instances` only support ccs cluster, better to clarify that

Reply:
@xueli181114 thanks for highlighting this.

Is adding machine pools unavailable for OSD Red Hat cloud account clusters overall? In my testing the **Add machine pool** button is greyed out when I deploy OSD on AWS using the Red Hat cloud account model. Or, is that due to the quota that I have available? Based on the answer to that question, I will update the content accordingly.

@0kashi what is your view on this? Thanks!

Reply:
Adding a machine pool should be available for all clusters as long as you have available subscription.  In ROSA this is not a concern.  In OSD you'd need available subscription, and when creating it should limit you to what you have left.

Reply:
Thanks 0kashi. What is your view about the spot instance option and Red Hat cloud account OSD deployments. Is there ever a scenario where that is supported? Thanks.

Reply:
@xueli181114 and @0kashi I have updated the spot instance steps for OSD to reference CCS deployments only.

I have also updated all of the spot instance UI sections with substeps about selecting to use the default spot instance price or a maximum.

Lastly, I have added an important note to all of the spot instance sections, including in the ROSA CLI section, to state that spot instances should be used only for workloads that can tolerate interruptions.